data/src/main/scala/ch.epfl.scala.index.data/github/GithubDownload.scala

```diff
@@ -126,6 +129,80 @@ class GithubDownload(
       )
     )
   }
+  def update(
+      project: Project,
+      projectForm: ProjectForm,
+      fromStored: Boolean = false
+  ): Project = {
```

Comment:
This is not the right place because it does not download anything from github or reading/validating the github json from the filesystem.

I would put it in `Project` instead.

Reply:
we agreed to remove the update from github and do it later